### DISCUSSION TITLE
How to add additional classes when the target element contains some class?

### DISCUSSION THREAD

**Discussion by jokerwon** [tailwindlabs/tailwindcss#13283] [Help] [answered]
I wanna add some additional classes like `bg-black` to an element when it has a specific custom class name like `mini`.

eg. 
~~~vue
<div class="bg-white" :class="{ mini: size === 'mini' }" />
~~~

expect
~~~vue
<div class="bg-white  xxx-[.mini]:bg-black" :class="{ mini: size === 'mini' }" />
~~~

Is there any utility class helping me to achieve？

**Comment by andymccray** [comment]
If possible, I’d consider doing this at the template level. Pseudo-coding the conditional as I’m not 100% what templating language this is:

```html
<div class="bg-white {{ size === 'mini' ? 'bg-black' }}">…</div>
```

Alternatively you could have a CSS rule that matched elements that have both the `mini` and `bg-white` classes (or preferably some other identifier) but this seems more fragile to me.

Tailwind Play example: https://play.tailwindcss.com/d0K7ZdbJs5

**Reply by jokerwon** [reply, marked as answer]
Thanks for your answer. Similarly I achieve it by dynamic class in Vue.

```vue
<div class="bg-white" :class="size === 'mini' ? ['bg-white'] : []" />
```
